ext/phar/phar_object.c

```diff
@@ -4438,7 +4438,7 @@ PHP_METHOD(Phar, extractTo)
 							RETURN_THROWS();
 						case 0:
 							zend_throw_exception_ex(phar_ce_PharException, 0,
-								"Phar Error: attempted to extract non-existent file or directory \"%s\" from phar \"%s\"",
+								"phar error: attempted to extract non-existent file or directory \"%s\" from phar \"%s\"",
```

Comment:
It's because the vast majority of such errors say "phar error", and just some of them say "Phar error" or "Phar Error". I am all for getting rid of these prefixes, but I'd prefer revisiting the problem later (if you have no objection).

Reply:
Sure, that's fine.